src/pages/informationmodel-details-page/information-model-details-page.jsx

```diff
@@ -26,11 +26,18 @@ const renderJSONSchema = schema => {
 
   return (
     <div>
-      <pre style={{ whiteSpace: 'pre' }}>{JSON.stringify(schema, null, 2)}</pre>
+      <pre style={{ whiteSpace: 'pre-wrap', overflowWrap: 'break-word' }}>
+        {JSON.stringify(schema, null, 2)}
+      </pre>
     </div>
   );
 };
 
+const renderRdfData = rdfData => (
+  <pre style={{ whiteSpace: 'pre-wrap', overflowWrap: 'break-word' }}>
```

Comment:
Use rather a styled component

Reply:
Hehe, you know there's a better alternative :>